cfpb/consumerfinance.gov
# Update design-system to 0.4.0

## Changes

- Update design-system to 0.4.0 to bring in https://github.com/cfpb/design-system/pull/1080


## How to test this PR

1. Pull branch and build and visit and see that the two digit bulleted list on http://localhost:8000/administrative-adjudication-proceedings/ is not cropped. 


## Screenshots

Before:
<img width="778" alt="Screen Shot 2020-10-28 at 2 12 22 PM" src="https://user-images.githubusercontent.com/704760/97479016-2419f800-1928-11eb-94c8-a70061b4ca4b.png">

After:
<img width="764" alt="Screen Shot 2020-10-28 at 2 14 07 PM" src="https://user-images.githubusercontent.com/704760/97479038-2a0fd900-1928-11eb-9007-c1c17e8c04c5.png">
